Riot/Modules/Room/RoomViewController.m

```diff
@@ -176,7 +176,10 @@ @interface RoomViewController ()
     
     // Observe kAppDelegateNetworkStatusDidChangeNotification to handle network status change.
     id kAppDelegateNetworkStatusDidChangeNotificationObserver;
-    
+
+    // Observers to manage MXSession state (and sync errors)
+    id kMXSessionStateDidChangeObserver;
```

Comment:
`k` prefix is normally used for constants, prefer name like `sessionStateDidChangeObserver`.